XSLT/tslaqdctomods.xsl

```diff
@@ -58,14 +58,26 @@
                     <xsl:apply-templates select="dc:title/text()"/>
                 </title>
             </titleInfo>
-            <recordInfo>
-              <recordContentSource>Tennessee State Library &amp; Archives</recordContentSource></recordInfo>
             <accessCondition type="local rights statement">
               <xsl:apply-templates select="dc:rights"/>
             </accessCondition>
           <location>
             <xsl:apply-templates select="dc:identifier[starts-with(., 'http://')]"/>
           </location>
+          <!-- creator(s) -->
+          <xsl:apply-templates select="dc:creator"/>
+          <!-- description -->
+          <xsl:apply-templates select="dc:description"/>
+          <!-- date -->
+          <xsl:apply-templates select="dc:date"/>
+          <!-- spatial -->
+          <xsl:apply-templates select="dcterms:spatial"/>
+          <recordInfo>
+              <recordContentSource>Tennessee State Library &amp; Archives</recordContentSource>
+              <languageOfCataloging>
+                  <languageTerm term="code" authority="iso639-2b">eng</languageTerm>
```

Comment:
This isn't valid.  Should be:
`
<languageTerm type="code" authority="iso639-2b">eng</languageTerm>`

Reply:
Thanks for catching this - sorry for the copy and paste error.